packages/relay/tests/lib/eth.spec.ts

```diff
@@ -673,6 +755,164 @@ describe('Eth calls using MirrorNode', async function () {
     const result = await ethImpl.getTransactionByBlockNumberAndIndex(defaultBlock.number.toString(), defaultBlock.count);
     expect(result).to.equal(null);
   });
+
+  describe('eth_getLogs', async function () {
+
+    const expectLogData1 = (res) => {
```

Comment:
nit:  A few shared checks in here can be pulled out. 
```ts
const expectLogData = (res) => {
      expect(res.blockHash).to.eq(defaultDetailedContractResults.block_hash);
      expect(res.blockNumber).to.eq(defaultDetailedContractResults.block_number);
      expect(res.removed).to.eq(false);
      expect(res.topics).to.exist;
      expect(res.transactionHash).to.eq(defaultDetailedContractResults.hash);
      expect(res.transactionIndex).to.eq(defaultDetailedContractResults.transaction_index);
    };
```

Then each `expectLogDataX(res)` can call `expectLogData(res)`